modules/client/src/addressList/interfaces.ts

```diff
@@ -43,13 +44,13 @@ export interface IClientAddressListMethods extends IClientCore {
   getProposals: (
     params: IProposalQueryParams,
   ) => Promise<AddressListProposalListItem[]>;
-  getSettings: (pluginAddress: string) => Promise<IPluginSettings | null>;
```

Comment:
If we now have "voting" settings, as opposed to just settings, is there any other type of settings that might need a separate `getXXXSettings`?  If so, we might want to rename `getSettings` to `getVotingSettings`

Reply:
Nope, what I was thinking was to go back on the voting mode, right now it is handled with the booleans but maybe it makes mor sense to use an enum because its easier to handle and return

Reply:
My point was about the function name (getSettings). Wether to keep it with a generic name (when it only has part of the settings) or not.

Regarding the booleans/enum's, whatever is more comfortable for a developer. I think the App team will have two separate checkboxes but this may change at any time.

Reply:
Yeah the other settings are handled with `getMembers` for the addresslist and `getToken`for the tokenvoting, I dont plan on having a generic `getSettings`that resturns all settings, but maybe it makes changing it to `getVotingSettings` sense for the sake of clarity and consistency.

Reply:
About the enum think I believe I'm going to switch to enum, for me as a developer seems more straigthforward using and enum for things that cannot happen at the same time and booleans for the things that can happen, for example if `earlyExecution` and `voteReplacement` could be true at the same time this would make sense to be a boolean since this is not going to happen, imo it does not make sense